src/nwb_benchmarks/benchmarks/network_tracking_remote_file_reading.py

```diff
@@ -0,0 +1,86 @@
+"""Basic benchmarks for stream NWB files and their contents."""
+
+import os
+
+from nwb_benchmarks.core import (
+    get_s3_url,
+    network_activity_tracker,
+    read_hdf5_fsspec_no_cache,
+    read_hdf5_nwbfile_fsspec_no_cache,
+    read_hdf5_nwbfile_remfile,
+    read_hdf5_nwbfile_ros3,
+    read_hdf5_remfile,
+    read_hdf5_ros3,
+)
+
+TSHARK_PATH = os.environ.get("TSHARK_PATH", None)
```

Comment:
I think it would be useful to move `TSHARK_PATH = os.environ.get("TSHARK_PATH", None)` to `core/__init__.py` and import it instead. In this way the tests don't need to know about the `"TSHARK_PATH"` environment variable, and if we need in more than one place it becomes easier to change later.

Reply:
The same also appears in [/network_tracking_remote_slicing.py](https://github.com/NeurodataWithoutBorders/nwb_benchmarks/pull/21/files#diff-ca1bc7c7c1eed4a5c3e878ebdd6df6a14069b8fc0ee3a4dd258f3891da063a3d)

Reply:
Good point; moved it a level even higher since it might be needed by anything else in the repo